app/controllers/staff/pages_controller.rb

```diff
@@ -30,6 +30,12 @@ def update
     end
   end
 
+  def publish
+    @page.update(published_body: @page.unpublished_body)
```

Comment:
My only question is about the publishing process: why copy from an unpublished body field to a published one, rather than just have a single body field and a Boolean "published?" Column, default: false?

Reply:
@adarsh This will likely make more sense in the next PR. The idea is that the person editing will be able to preview changes to the body before they get published.